op-challenger/game/fault/preimages/large.go

```diff
@@ -23,23 +23,13 @@ var errNotSupported = errors.New("not supported")
 
 var _ PreimageUploader = (*LargePreimageUploader)(nil)
 
-// MaxLeafsPerChunk is the maximum number of leafs per chunk.
-const MaxLeafsPerChunk = 300
+// MaxBlocksPerChunk is the maximum number of keccak blocks per chunk.
+const MaxBlocksPerChunk = 300
 
 // MaxChunkSize is the maximum size of a preimage chunk in bytes.
 // Notice, the max chunk size must be a multiple of the leaf size.
 // The max chunk size is roughly 0.04MB to avoid memory expansion.
-const MaxChunkSize = MaxLeafsPerChunk * matrix.LeafSize
-
-// Chunk is a contigous segment of preimage data.
-type Chunk struct {
-	// Input is the preimage data.
-	Input []byte
-	// Commitments are the keccak commitments for each leaf in the chunk.
-	Commitments [][32]byte
-	// Finalize indicates whether the chunk is the final chunk.
-	Finalize bool
-}
+const MaxChunkSize = MaxBlocksPerChunk * matrix.LeafSize
```

Comment:
Here is another place where I think moving the `matrix.LeafSize` to a `keccakTypes.BlockSize` makes more sense, since unit elimination would look like:

`MaxChunkSize` = `# blocks / chunk` * `# bytes / bock` which simplifies to `# bytes / chunk`.

Reply:
Rename/move applied here - I'm not sure if you're suggesting further changes to this line. Agreed it reads better to consistently talk about blocks rather than a mix of block and leaf.